packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -9060,89 +9057,6 @@ Map {
     "render": [Function],
   },
   "TreeView" => Object {
-    "TreeNode": Object {
-      "$$typeof": Symbol(react.forward_ref),
-      "propTypes": Object {
-        "active": Object {
```

Comment:
Once `TreeView.TreeNode` is added back in, regenerate snapshots with `yarn test -u` and this change should be reverted/go away.